02-household-queries/qt_summaries-gpt-3.5-turbo-instruct-k_4.json

```diff
@@ -0,0 +1,851 @@
+[
+    {
+        "id": 1,
+        "question": "The client has her 19 year old grandchild living with her. She says she does her own thing and they don't really share food. Does the grandchild need to be included?",
+        "guru_cards": [
+            "Who are mandatory HH members for food stamps?"
+        ],
+        "recall": 1.0,
+        "extra_cards": 3,
+        "derived_questions": [
+            "Does the grandchild live with you?",
+            "Is the grandchild financially dependent on you?",
+            "Do you provide regular support for the grandchild?"
+        ],
+        "all_retrieved_cards": {
+            "Can children under 18 years of age ever be certified as a separate household from other adults living in the home?": {
+                "score_sum": 0.7314764598621692,
+                "derived_questions": [
+                    "Does the grandchild live with you?",
+                    "Do you provide regular support for the grandchild?",
+                    "Is the grandchild financially dependent on you?"
+                ],
+                "entire_card": "Can children under 18 years of age ever be certified as a separate household from other adults living in the home?\nAnswer:\nYes, if the adults with whom they are living do not exercise parental control or offer financial support. Clients who are officially emancipated usually fall under this category. If the client not entirely certain or it's close but messy we can error on the side of applying and inform the client the county will help them figure out the household if need be.\nExample: Ms. G is 17 years old. She lives with her aunt. Ms. G works and purchases and prepares her own meals. When questioned about household composition she stated that her aunt does not exercise parental control over her. The case manager certifies Ms. G as a one-person household.\nFood stamps only",
+                "summary": "Children under 18 years of age can be certified as a separate household from other adults living in the home if the adults do not exercise parental control or offer financial support, and if the child is officially emancipated."
```

Comment:
**Relevance to the question**: 🟡 Potentially relevant
- The original question specifies that the client has a 19yo grandchild. Based on the retrieved Guru card, the summary describes the rules related to children _under_ 18yo being certified as a separate household. The user would need to reason whether to infer that the same rules apply to children over 18yo, or seek a different answer. 
**Correctness**: 🟡 Mixed (partially correct, partially incorrect)
- The summary offers 3 criteria for when a child under 18 can be certified as a separate household, the first two of which are correct (other adults do not `exercise parental control` or `offer financial support`), but the third (child is `officially emancipated`) is a mistake that interprets an example in the Guru card, "Clients who are officially emancipated usually fall under this category", as a rule. 
**Helpfulness**: 🔴 Potentially harmful (leads the user down the wrong path)
- The summary implies that official emancipation is a requirement for listing a child as a separate household member
- The summary describes rules for children under 18 when the question is about a grandchild over 18

Reply:
Rating scale: 

**Rating: Relevance to the question**:
- 🟢 Directly relevant 
- 🟡 Partially relevant 
- 🟡 Potentially relevant (seems related to the question but requires reasoning to make the connection)
- 🔴 Irrelevant

**Rating: Correctness as a summary of the Guru card**:
- 🟢 Correct (100% correct)
- 🟡 Mixed (partially correct, partially incorrect)
- 🟡 Missing Info (100% correct info present but missing some crucial piece of info)
- 🟡 Irrelevant info (100% correct info present but also a lot of additional neutral info present, e.g., definitions of concepts in the response that are not required to answer the user's actual question)
- 🔴 Incorrect (100% incorrect)

**Rating: Helpfulness**
- 🟢 Helpful
- 🟡 Neutral (technically not wrong answer but not really helpful in the context of the question)
- 🟠 Unhelpful (confusing or obviously wrong answer, but unlikely to result in the user going down the wrong path)
- 🔴 Potentially harmful (leads the user down the wrong path for benefits application, is emotionally triggering, etc.)